packages/trader/src/Stores/Modules/Trading/Helpers/multiplier.js

```diff
@@ -27,14 +25,3 @@ export const onChangeCancellationDuration = ({ event, onChangeMultiple }) => {
         cancellation_duration: value,
     });
 };
-
-// eslint-disable-next-line no-empty-pattern
-export const connectWithContractUpdate = connect(({}, { contract = {} }) => {
```

Comment:
@jim-deriv 🙂 since you are removing connectWithContractUpdate(), please also remove it from components that are using it: ContractCardBody, AccumulatorCardBody, MultiplierCardBody, SummaryCard, ToggleCardDialog

Reply:
Hi @maryia-deriv, I've explained this in the slack group conversation.